src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.domain
+
+class LottoMachine(private val lottoFactory: LottoFactory) {
+
+    fun buyAuto(cost: Int): List<Lotto> {
+        val count = cost / LOTTO_COST
+        return (1..count).map { lottoFactory.create() }
```

Comment:
이렇게도 작성해 볼 수 있습니다 :)  
```suggestion
  return List(count) {}
```